03-viz.Rmd

```diff
@@ -290,6 +290,151 @@ faithful_scatter <- ggplot(faithful, aes(x = waiting, y = eruptions)) +
 faithful_scatter
 ```
 
+
+### The Canadian languages data set
+
+We will return to the `can_lang` data set, which contains counts of languages from the 2016 Canadian census.
```

Comment:
We should cite the census and `canlang` again here in case anyone jumps into reading the textbook here. We should also formally cite all the data in this chapter.